test/parsers/html-parser.js

```diff
@@ -4,14 +4,18 @@ const YAML = require('yaml');
 const fs = require('fs');
 const path = require('path');
 const CONFIG_FILE = '../resources/config.yaml';
+const NJK_CONFIG_FILE = '../resources/config-njk.yaml';
 
 describe('HtmlParser', function () {
   {
     before(function () {
       const file = fs.readFileSync(path.resolve(__dirname, CONFIG_FILE), 'utf8');
       const yamlData = YAML.parse(file);
-
       this.output = parseHTML(yamlData.paths['/movies/{actor}']['get'].responses, '', '');
+
+      const njkFile = fs.readFileSync(path.resolve(__dirname, NJK_CONFIG_FILE), 'utf8');
```

Comment:
Do other tests also need this?

Reply:
No, do I load this inside of the specific test?

Reply:
That would be best then yes.